### DISCUSSION TITLE
Application under review

### DISCUSSION THREAD

**Discussion by kernelpanic77** [github/india#620] [Students]
It's been more than 48 hours, but my dashboard still shows that my application is under review. 
Is this happening with everyone ?

**Comment by Devesh21700Kumar** [comment]
The application is still under review. I guess that it will be done by the end of the day.

**Comment by madrigal1** [comment]
its been 3 days for me no change in status

**Reply by Devesh21700Kumar** [reply]
Is it still under review?

**Reply by madrigal1** [reply]
yes,still shows under review no change been more more than a week

**Comment by Devesh21700Kumar** [comment]
> It's been more than 48 hours, but my dashboard still shows that my application is under review. Is this happening with everyone ?

Is it still under review?

**Reply by kernelpanic77** [reply]
yeah

**Comment by punitjain03** [comment]
Hi , Currently we are reviewing the profiles, From today EOD or tomorrow morning, We will notify the status of the application status. Post that you can apply for the organisations.
